test/xref2/lib/common.cppo.ml

```diff
@@ -2,15 +2,14 @@ open Result
 
 (* Example usage of these:
 
-$ dune utop src/xref2/test/lib
+$ dune utop test/xref2/lib
 utop # open Odoc_xref2;;
 utop # open Odoc_xref_test;;
 utop # let test_data = "module type M = sig type t end module N : M type u = N.t";;
-utop # let id, docs, sg = Common.model_of_string test_data;;
+utop # let id, sg, docs = Common.model_of_string test_data;;
```

Comment:
I think the last value in this triple is now the optional 'canonical' value - `model_of_string` ends up calling `read_interface`:

```ocaml
let read_interface root name intf =
  let id = `Root (root, Odoc_model.Names.ModuleName.make_std name) in
  let sg, canonical =
    read_signature Odoc_model.Semantics.Expect_canonical Env.empty id intf
  in
  (id, sg, (canonical :> Odoc_model.Paths.Path.Module.t option))
```

Reply:
Indeed, so we probably need to rename the variable from `docs` to `canonical`. (Sorry for the delay to respond)